framework/src/utils/MatrixTools.C

```diff
@@ -31,7 +31,7 @@ int inverse(const std::vector<std::vector<Real> > & m, std::vector<std::vector<R
     for (const auto & matrix_entry : rowvec)
       A.push_back(matrix_entry);
 
-  int error = inverse(A, n);
+  int error = inverse(A, (int) n);
```

Comment:
This does not seem the right thing. Why doesn't that method take an unsigned int?